QA_Nano.md

```diff
@@ -0,0 +1,107 @@
+
+### Creation
+- Click "Create" and add an incorrect PIN on the modal: the "Invalid PIN" error should appear
+- Insert the correct pin and click "Go".
+- After a while, the NC should be created.
+- A screen should be shown with the details of the nano contract. Copy its identifier for later tests.
+- The three methods `bet`, `set_result` and `withdraw` should be shown
+
+# Interacting with a Nano Contract
+This first interaction should happen before the "Date of Last Bet", so that the bet is valid.
+
+### Betting - Method Parameters
+- On the Nano Contracts details screen, "Available Methods" section, click on `bet`
+- Without inserting anything, click "Execute Method". All fields should become red.
+- Insert an invalid address on the "Address" field and click "Execute". Should stay red.
+- Insert a valid address from another wallet and click "Execute". Should become green.
+- For testing purposes, insert an address from this wallet to continue the next steps.
+- Insert `1` on the "Score" field and click "Execute"
+- The PIN should be requested before any validation on the actions is made.
+- Insert the correct pin and send the transaction. Check that it becomes voided because the action was missing.
+
+### Betting - Action
+- Select the `bet` method and fill all the fields again. Take note of the address used. 
+- Select "Deposit" from the "Type" options
+- Click "Execute" and check that the other fields become red
+- Select any token and the field immediately turns green
+- Insert any amount and the field immediately turns green too
+- Click "Remove" and check that the whole row is removed.
+- Create an action with the deposit of `0.01` HTR and click Execute.
+- Now the Token Details screen should indicate the field `total` is `0.01`
+- The "Balances" section should indicate a balance of `0.01` for token `00`
+
+### Late betting
+- We need a different valid bet to decide a winner, so create another bet with:
+  - Another address from this same wallet
+  - Score: `2`
+  - Deposit value: `0.02` HTR
+- The "Balances" section of the contract should indicate `0.03` for token `00`
+- Now wait for a while until the timestamp informed for `date_last_bet`
+- Create a new valid bet, and click "Execute". A message will appear informing of an error, and the transaction will not be processed.
+
+### Setting Results
+- Now, on "Nano Contract Detail" screen, select `set_result`
+- Without filling any fields, click "Execute". The "result" field should become red.
+- Enter any character in it and it should become green immediately
+- Click "Select Address to Sign" and select the same address used as "Address to Sign" on initialization
+- Click on "Sign Data" without filling any fields. They should become red.
+- Insert `2` on "Data to Sign" and fill the wrong pin, then click "Sign Data". An "Invalid PIN" error should appear immediately
```

Comment:
In the other places you continue as if the result is `1`

Reply:
Thanks for the catch! Fixed it.